src/Lab/Experiments/VulkanTriangle/HelloTriangleApplication.cs

```diff
@@ -257,13 +270,40 @@ private unsafe void CleanupSwapchain()
             _vkSwapchain.DestroySwapchain(_device, _swapchain, null);
         }
 
+        private unsafe string[] GetOptimalValidationLayers()
```

Comment:
nit: nullables (don't think it's enabled in this project but doesn't hurt to annotate anyway)

```suggestion
        private unsafe string[]? GetOptimalValidationLayers()
```